.github/workflows/conda-lock.yml

```diff
@@ -12,7 +12,6 @@ jobs:
     strategy:
       matrix:
         python-version:
-          - "3.9"
```

Comment:
Removed this because we can't build the docs deps for 3.9 (because of limitations in plum [a dependency of `quartodoc`]).